guides/taxation/overview-of-taxation.md

```diff
@@ -0,0 +1,149 @@
+  the price (for value-added taxes) or added to the price (U.S. taxes).
+
+Solidus calculates tax based on the matching tax rate(s) for the order's [tax
+address](#tax-addresses). 
+
+## Tax addresses
+
+If an order's `tax_address` falls within a specific zone, the tax rates that you
+have configured would apply for all the line items and shipments in that zone
+that have a matching tax category.
+
+By default, the `tax_address` used for orders is the customer's shipping
+address. This is how most tax jurisdictions require taxes to be calculated. 
+However, you can configure your store to globally use customer billing
+addresses instead in any initializer file inside the `config/initializers/`
+directory:
+
+```ruby
+Spree::Config[:tax_using_ship_address] = true
+```
+
+## Sales tax and value-added tax
+
+In ecommerce, [consumption tax][consumption-tax] either takes the form of sales
+tax or value-added tax. Some countries use other names for their taxes, but
+generally all modern consumption taxes would be considered one of these two
+types.
+
+Solidus's models support both types of taxes. In the case of a product or
+shipment:
+
+- **Sales tax** is calculated as _additional_ taxes on top of the listed price.
+  (U.S.-style taxation.)
+- **Value-added tax (VAT)** is calculated as _included_ in the listed price.
+	Solidus lists all VAT amounts below the item total on checkout summary pages.
+	For more information about VAT, see [Value-added tax (VAT)][vat].
+
+[vat]: value-added-tax.md
+
+### Tax in the United States
+
+Note that sales tax in the United States can get exceptionally complex. Each
+state, county, and municipality might have a different tax rate.
+
+<!-- TODO:
+  Create and link to an article that's all about United States taxes.
+-->
+
+If you intend to ship products between states, and your store is based in the
+United States, we recommend that you use an external service like
+[Avatax][avatax] or [Tax Cloud][tax-cloud] to automate your U.S. tax rates.
+Solidus has extensions for both of these services:
+[`solidus_avatax_certified`][solidus-avatax-certified] and
+[`solidus_tax_cloud`][solidus-tax-cloud].
+
+[avatax]: https://www.avalara.com/
+[tax-cloud]: https://taxcloud.net/
+[solidus-avatax-certified]: https://github.com/boomerdigital/solidus_avatax_certified
+[solidus-tax-cloud]: https://github.com/solidusio-contrib/solidus_tax_cloud
+
+## Order taxation
+
+Once the order has a `tax_address` specified, tax can be calculated for all of
+the line items and shipments associated with a `Spree::Order`:
+
+Note that any promotional adjustments are applied before tax adjustments. This
+is to comply with tax regulations for value-added taxation [as outlined by the
+Government of the United Kingdom][uk-vat-discounts] and for sales tax [as
+outlined by the California State Board of Equalization][ca-tax-discounts].
+
+1. Solidus iterates over all of the order's `Spree::LineItem`s. It selects the
```

Comment:
I think the class name does not help that much here. Maybe `order's line items` is good enough?